clusters/ptl/base/kustomization.yaml

```diff
@@ -16,3 +16,4 @@ patches:
     kind: Kustomization
     annotationSelector: hmcts.github.com/kustomize-defaults != disabled
 - path: ../../../apps/admin/ptl/base/kustomize.yaml
+- path: ../../../apps/admin/traefik-crds-upgrade-v26/kustomize.yaml
```

Comment:
Don't forsee it causing issues since sbox worked fine but might be best to do PTL as part of prod just because of Jenkins and ADO. Ptlsbox hasn't been tested yet. Can't find any info on how we split out PTL last time round.